### ISSUE TITLE
Can I disable spawn chunks with this mod?

### ISSUE THREAD

**Issue by mrGRINCH** [Dutchs/ConfigurableSpawnChunks#1]
Am I able to disable the permanently loaded spawn chunks with this mod?

**Comment by Dutchs** [comment]
No, this mod just lets you join the game prior to all the forced chunks having been generated.
It speeds up how fast you join but these chunks are still force loaded by minecraft/forge.
